bgpd/bgp_zebra.c

```diff
@@ -1904,7 +1904,11 @@ void bgp_zebra_terminate_radv(struct bgp *bgp, struct peer *peer)
 		zlog_debug("%u: Terminating RA for peer %s", bgp->vrf_id,
 			   peer->host);
 
-	zclient_send_interface_radv_req(zclient, bgp->vrf_id, peer->ifp, 0, 0);
+	if (peer->ifp)
```

Comment:
Don,

The code calls three different functions.

1) bgp_nht_dereg_enhe_capability_intfs 
2 Call zclient_send_interface_radv_req with false as one of the input() directly.
3) bgp_zebra_terminate_radv()

Can you explain a bit on when each of these function gets called ?

Currently, I see little inconsistency with respect to calling each of these functions.

Reply:
I see your point.  It may be I was too hasty once I had a solution that worked and didn't try to make it consistent.  I'll revisit and update the PR with any changes (and explanations.)